src/DistanceCalculatorAVX.cpp

```diff
@@ -364,11 +365,16 @@ double DistanceCalculator::newCalcDNA(int a, int b) {
         for (int j = 0; i < numOfInts && j < 31;
              i += 8) { // a maximum of 32 vectors allowed not to overflow the
                        // int the numbers are stored in
-            seq1 = *(__m256i *)&Achar[i];
-            seq2 = *(__m256i *)&Bchar[i];
 
-            gap1 = *(__m256i *)&Agap[i];
-            gap2 = *(__m256i *)&Bgap[i];
+            seq1 = _mm256_load_si256((__m256i*)&Achar[i]);
+            //seq1 = *(__m256i *)&Achar[i];
```

Comment:
Why'd you leave these commented? Can we just delete them?

Reply:
I didn't have a firm reason, other than I suspect we're going to have to make a lot of similar changes through the codebase and I find it helpful having the before and after when comparing to other areas... we don't need to keep them in, though!